.github/workflows/_python-tests.yml

```diff
@@ -138,7 +153,6 @@ jobs:
                     "chromadb/test/property/test_filtering.py",
                     "chromadb/test/property/test_embeddings.py",
                     "chromadb/test/property/test_collections_with_database_tenant.py",
-                    "chromadb/test/property/test_collections_with_database_tenant_overwrite.py",
```

Comment:
why remove this?

Reply:
the test requires a server that supports auth

Reply:
after looking at it again, I'm not completely sure what its purpose is
we already have a test that checks tenant isolation: `chromadb/test/property/test_collections_with_database_tenant.py`

Reply:
the point of this test is to validate that the python client can handle overriding the tenant and database from an api key if its possible. This is important for UX

Reply:
ok, so could we delete this and add a new test similar to chromadb/test/client/test_cloud_client.py that tests providing an API token, database, and tenant?

Reply:
adding the fakes required to set up a test like that is enough work that it might make sense to just implement default auth methods on the Rust side, what do you think?